src/js/vre/chapter36/config/form.js

```diff
@@ -88,30 +99,72 @@ const formConfig = {
               }
             }
           }
+        },
+        nonVeteranInformation: {
+          title: 'Applicant information',
+          path: 'non-veteran-information',
+          depends: {
+            'view:isVeteran': false
+          },
+          initialData: {},
+          uiSchema: {
+            applicantSocialSecurityNumber: ssnUI,
+            applicantGender: {
+              'ui:title': 'Gender',
+              'ui:widget': 'radio'
+            },
+            seekingRestorativeTraining: {
+              'ui:title': 'Are you a child who is at least 14 years old, a spouse, or a surviving spouse with a disability and looking for special restorative training?',
+              'ui:widget': 'yesNo'
+            },
+            seekingVocationalTraining: {
+              'ui:title': 'Are you a child, a spouse, or a surviving spouse with a disability and looking for special vocational training',
+              'ui:widget': 'yesNo'
+            },
+            receivedPamphlet: {
+              'ui:title': 'Have you received a pamphlet explaining survivors\' and dependents\' educational assistance benefits?',
+              'ui:widget': 'yesNo'
+            }
+          },
+          schema: {
+            type: 'object',
+            properties: {
+              applicantSocialSecurityNumber,
+              applicantGender: {
```

Comment:
We try to use the properties from the schema as much as possible. In this case, you could import `applicantGender` from the schema properties or `gender` from the schema definitions and use it here. If you do the former, you'll have to put the `gender` definition in `defaultDefinitions`. 

Then, to set the labels, you can put them in the uiSchema for this field:

```js
applicantGender: {
  'ui:title': 'Gender',
  'ui:widget': 'radio',
  'ui:options': {
    labels: genders
  }
}
```

Our select and radio widgets support getting labels both from here and in the `enumNames` property of the schema.